middleware/src/system/system_test.go

```diff
@@ -17,6 +17,7 @@ func TestSystem(t *testing.T) {
 	argumentMap["network"] = "testnet"
 	argumentMap["bbbConfigScript"] = "/home/bitcoin/config-script.sh"
 	argumentMap["bbbCmdScript"] = "/home/bitcoin/cmd-script.sh"
+	argumentMap["bbbSystemctlScript"] = "/home/bitcoin/bbb-systemctl.sh"
```

Comment:
What are these paths? Is this the specific regtest setup? On the Base, there is no `/home/bitcoin` directory and these files are called via symlinks in `/usr/local/sbin/`

Reply:
That are only test values to test the Getters. Could be a random string. might change them to avoid confusion.